state-chain/amm/src/common.rs

```diff
@@ -266,6 +271,7 @@ pub(super) trait SwapDirection {
 
 	/// Returns the equivalent saturated amount in the output asset to a given amount of the input
 	/// asset at a specific tick, will return None iff the tick is invalid
+	#[allow(dead_code)]
```

Comment:
Yeah think we can remove this - was previously used in collect and burn limit functions, but we don't use it anymore, so let's delete it.